scripts/analysisTools/plotUtils/utility.py

```diff
@@ -22,6 +22,7 @@
                  "Top"        : ROOT.kGreen+2,
```

Comment:
Can you not use the datagroups option for this as well? ROOT accepts a hex color---SetColor(ROOT.TColor("#...")) I think. It would be kind of nice to at least have the coolers and labels the same even if we make plots from different places.

Reply:
I guess I can, this was just what I used to have for my plots in the past. But right now the groups don't use hex color IDs, so I will have to change them too. Let's keep it for next time